ouroboros-consensus/changelog.d/20230525_172337_coot_remove_on_framework.md

```diff
@@ -0,0 +1,4 @@
+### Breaking
+
+* Removed `ConnectionId` `Condence` instance.
```

Comment:
```suggestion
* Removed `ConnectionId` `Condense` instance.
```

Reply:
I should have accepted your original suggestion :grin: